app/routes/events/components/EventEditor/index.tsx

```diff
@@ -91,9 +91,10 @@ const validate = createValidator({
   title: [required('Tittel er påkrevd')],
   description: [required('Kalenderbeskrivelse er påkrevd')],
   eventType: [required('Arrangementstype er påkrevd')],
-  location: [
-    atLeastOneFieldRequired(
-      ['location', 'mazemapPoi'],
+  location: [required('Sted er påkrevd')],
```

Comment:
I might be wrong - but isn't `location` empty if mazemap is used? And thus ideally speaking not required

Reply:
Yes you're right! I just copied the existing validation when refactoring, so this is actually a bug in prod. However, it is rarely caught as `event.location` has a default value of `TBA`. 

https://github.com/webkom/lego-webapp/pull/4131/commits/52cb725b459ab8fdecd1eb9590cb62c408cadbf6#diff-9a787b7f7873cbf26136947b17f0a926272d1ce799f81ac1b7c0b0bab7f5c4efL707-L709